lib/client/api.go

```diff
@@ -2763,6 +2763,8 @@ func makeProxySSHClient(ctx context.Context, tc *TeleportClient, sshConfig *ssh.
 		log.Infof("Connecting to proxy=%v login=%q using TLS Routing", tc.Config.WebProxyAddr, sshConfig.User)
 		c, err := makeProxySSHClientWithTLSWrapper(ctx, tc, sshConfig, tc.Config.WebProxyAddr)
 		if err != nil {
+			fmt.Fprintf(os.Stderr, "Unable to successfully connect and authenticate to ssh proxy address=%v. Required for Teleport client.  Confirm connectivity and availability.", tc.Config.WebProxyAddr)
+			fmt.Fprintln(os.Stderr)
 			return nil, trace.Wrap(err)
```

Comment:
Is it OK if we add the details to the `trace.Wrap` instead of printing directly to stderr? I'd rather not print to the console in a lib package.

Also, could we push the message into a small helper method and reuse it throughout?

Eg:

```go
			return nil, trace.Wrap(err, confirmConnectivityErrorMsg(tc.Config.WebProxyAddr))
```

```go
func confirmConnectivityErrorMsg(proxyAddr string) string {
	return fmt.Sprintf("" + 
		"Unable to successfully connect and authenticate to ssh proxy address=%v. " + 
		"Required for Teleport client. " +
		"Confirm connectivity and availability.")
}
```

Reply:
thanks @codingllama , moved to helper function and using